actions.yaml

```diff
@@ -0,0 +1,51 @@
+get-identity:
+  description: Get a user using either the identity ID or the user email.
+  params:
+    identity-id:
+      description: The Identity ID
+      type: string
+    email:
+      description: The user's email
+      type: string
+delete-identity:
+  description: Delete a user using either the identity ID or the user email.
+  params:
+    identity-id:
+      description: The Identity ID
+      type: string
+    email:
+      description: The user's email
+      type: string
+create-admin-account:
+  description: |
+    Create an admin user. If no password was provided, the command will return a
+    magic link where the user will be able to set their password.
+  params:
+    username:
+      description: The admin username
+      type: string
+    email:
+      description: |
+        The admin's email, this email must not be associated with any other account
+        (user or admin)
+      type: string
+    password:
+      description: The admin's password
+      type: string
+    name:
+      description: The admin's name
+      type: string
+    phone_number:
+      description: The admin's phone number
+      type: string
+  # TODO: Remove required password once password resetting is implemented
+  required: ["username", "password"]
+run-migration:
```

Comment:
Should this not be done automatically as part of the upgrade process?

Reply:
I think this was the data platform team's suggestion for handling migrations. 

It may be a breaking migration, so the admin will have to first take a backup. In addition, if the charm tries to run a migration and it fails for some reason, the admin would need to be able to trigger it again (e.g. with bigger timeout or after some transient issue has been resolved).

Reply:
This is *sort of the point* of operators, though, precisely to automate this kind of stuff

Sure, the upgrade guide might say:

- Run your DB backup
- Refresh the Kratos charm

But I still think this should be handled automatically

Reply:
I don't disagree with this, but we chose this approach as per the data platform team's suggestion (as I mentioned before). 

Having said that, it is out of scope for this PR to handle the db migration logic. We will create a task in our backlog for investigating this in the future.